requirements.txt

```diff
@@ -9,8 +9,8 @@
 # that are dependencies of libraries listed in the next section
 
 ga4gh-common==0.0.5
-ga4gh-schemas==0.0.8
-ga4gh-client==0.0.3
+ga4gh-schemas
```

Comment:
@dcolligan It should be possible to pin the version in the requirements.txt for released versions. But since the schemas aren't updated in pypi with every git commit, we need a better way to manage working against the master branch of schemas, which in the past was maintained by careful runs of `process_schemas.py`. 

This is modeled in a way with what is done with the schemas, which use github repos as a dependency via a git tag. The compliance/master runs against schemas/master, so if schemas/master changes compliance breaks. This is not currently the case with the server, which this will change.

So I would say you're both right! For released versions I think we will select specific versions in the requirements and remove the constraints.txt. The master-development versions point at the running master branch of their dependency.